mne/io/edf/tests/test_edf.py

```diff
@@ -27,12 +27,12 @@
 from mne.io.edf.edf import _read_annotations_edf
 from mne.io.edf.edf import _read_ch
 from mne.io.edf.edf import _parse_prefilter_string
-from mne.io.pick import channel_indices_by_type
+from mne.io.pick import channel_indices_by_type, get_channel_type_constants
 from mne.annotations import events_from_annotations, read_annotations
-from mne.io.meas_info import _kind_dict as _KIND_DICT
 
 
 FILE = inspect.getfile(inspect.currentframe())
+KIND_DICT = get_channel_type_constants()
```

Comment:
I'd rather nest this in `test_edf_stim_ch_pick_up` -- if we ever inadvertently break `get_channel_type_constants`, it will break tests during the collection rather than the running phase, which is not so nice.